### PR TITLE
Create SparePart class and test for it

### LINKED ISSUE #11: Implement class for a spare part
We don't know what we need from this class yet, but at first we need some class that may describe common states of a spare part of a computer or printer: name, cost, repair cost, change cost. In the future we will add here methods that we would need

### PR DESCRIPTION
**Pull request by it-cio**
Close #11

### CODE REVIEW

File: tests/test_spare_parts.py [hunk 1]
```diff
@@ -22,4 +22,4 @@ def test_add_part():
 
 def test_to_json():
     part = SparePart('Drum Unit for LaserJet 1020')
-    assert part.to_json() == '{"name": "Drum Unit for LaserJet 1020", "parts": []}'
+    assert part.to_json() == '{"name": "Drum Unit for LaserJet 1020", "parts": ["Drum Unit for LaserJet 1020"]}'
```
Comment (Lehersun) [inline tests/test_spare_parts.py]: The internal structure of parts array after json serialisation must contain the same fields like name and parts

Comment (it-cio) [reply tests/test_spare_parts.py]: I'm reworked test  to assert that to_json method return correct json type

**Comment by Lehersun** [comment]
Don't forget about right naming of commits. Hotfix and update main are truly bad names. Names have to contain small description of included changes. If you'll look at this after a time, can you tell what was inside

**Comment by it-cio** [comment]
> 

ok, i understud

File: classes/SparePart.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            self.__parts.append(name)
+        else:
+            raise ClassInitializationError('Name must be string')
+
+    def get_parts_info(self):
+        parts = self.__parts
+        return parts
+
+    def add_part(self, parts):
+        if isinstance(parts, list):
```
Comment (Lehersun) [inline classes/SparePart.py:20]: add_part func must check if list parts contains objects instantiated from SparePart

File: tests/test_spare_parts.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+def test_exception_if_spare_part_name_not_a_string():
+    with pytest.raises(Exception) as ex:
+        SparePart([])
+    assert ex.type == ClassInitializationError
+
+
+def test_add_part():
+    part = SparePart('Feed Drive for LaserJet 1020')
+    parts_list = ['Repair kit for LaserJet 1020', 'Fuser for LaserJet 1020', 'Main Motor for LaserJet 1020']
```
Comment (Lehersun) [inline tests/test_spare_parts.py]: parts_list should be a list of objects SparePart

**Review by Lehersun — APPROVED** [review]

**Pull request merged**